src/components/dialogHelper/dialogHelper.js

```diff
@@ -172,7 +172,7 @@ define(['appRouter', 'focusManager', 'browser', 'layoutManager', 'inputManager',
     }
 
     function isHistoryEnabled(dlg) {
-        return dlg.getAttribute('data-history') === 'true';
+        return dlg.getAttribute('data-history') === 'false';
```

Comment:
wtf? :)

Reply:
What's wrong with this?

Reply:
> What's wrong with this?

logic reverse

Reply:
https://github.com/jellyfin/jellyfin-web/pull/871/commits/4fca1c15638e9f0cd4fc8317fe7d8baef115da66